src/preprocessing/point_in_poly/hierarchical_winding.jl

```diff
@@ -0,0 +1,238 @@
+# This bounding box is used for the hierarchical evaluation of the winding number.
```

Comment:
```suggestion
# This bounding box is used for the hierarchical evaluation of the `WindingNumberJacobsen`.
```
Make it clear to everyone that this is only used for the Jacobsen winding number.